relational_operators/TextScanOperator.cpp

```diff
@@ -288,7 +288,7 @@ void TextScanWorkOrder::execute() {
 
     std::fclose(file);
   } else {
-    BlobReference blob = storage_manager_->getBlob(text_blob_);
+    MutableBlobReference blob = storage_manager_->getBlobMutable(text_blob_);
```

Comment:
Actually, you're right. Although `release` is not a `const` method, as long as `blob` is not `const`, we are fine.

In other words, we don't need to change to `MutableBlobReference`. I'll fix it in another PR.